Apps/Common/src/Services/IEmailQueueService.cs

```diff
@@ -71,37 +71,22 @@ public interface IEmailQueueService
         /// <param name="shouldCommit">If true, the record will be written to the DB immediately.</param>
         void CloneAndQueue(Guid emailId, bool shouldCommit = true);
 
-        /// <summary>
-        /// Queues a new email invite.
-        /// Template will be looked up in the DB.
-        /// A new invite email will be added to the database.
-        /// </summary>
-        /// <param name="hdid">The users hdid to associate the invite with.</param>
-        /// <param name="toEmail">The To email address.</param>
-        /// <param name="activationHost">The host of the email validation endpoint.</param>
-        /// <param name="inviteKey">The email invite key to be sent.</param>
-        void QueueNewInviteEmail(string hdid, string toEmail, Uri activationHost, Guid inviteKey);
-
-        /// <summary>
-        /// Queues a new email using a populated Email object.
-        /// A new invite email will be added to the database.
-        /// </summary>
-        /// <param name="invite">The populated invite with associated email to send.</param>
-        void QueueNewInviteEmail(MessagingVerification invite);
-
-        /// <summary>
-        /// Queues an email to be sent.
-        /// </summary>
-        /// <param name="inviteEmailId">The id of the email to send.</param>
-        void QueueInviteEmail(Guid inviteEmailId);
-
```

Comment:
3 errors are being reported in CommonTests/Services/EmailQueueService_Test due to this change:

```
Error	CS1061	'IEmailQueueService' does not contain a definition for 'QueueNewInviteEmail' and no accessible extension method 'QueueNewInviteEmail' accepting a first argument of type 'IEmailQueueService' could be found
Error	CS1061	'IEmailQueueService' does not contain a definition for 'QueueInviteEmail' and no accessible extension method 'QueueInviteEmail' accepting a first argument of type 'IEmailQueueService' could be found
```

Reply:
fixed!